docs/installguide/advanced.rst

```diff
@@ -66,53 +67,76 @@ the rest of your environment, you can run::
 
 
 Portable tarballs / zip files with setup.py
-___________________________________________
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adequate to the online source, you can fetch a zip/tarball directly from
+`PyPi <https://pypi.python.org/pypi/ka-lite-static>`.
+Do this for the sake of carrying KA Lite on an offline media for use on systems
+that are not capable of installing the .exe/.dmg/.deb formats.
+
+A .tar.gz or .zip is unpacked and from command line inside the unpacked
+directory, run::
+
+    $> sudo python setup.py install.
+
+Beware that the PyPi sources do not contain assessment items, you need to
+`download assessment.zip manually <http://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip>`_ (~350 MB).
+
+
+Specific system setups
+______________________
+
+.. _raspberry-pi-wifi:
 
-You can also fetch a tarball directly from `PyPi <https://pypi.python.org/pypi/ka-lite-static>`.
-Do this for the sake of carrying KA Lite on an offline media. You can then
-unpack the tarball and run ``python setup.py install``.
+Raspberry Pi Wi-Fi
+~~~~~~~~~~~~~~~~~~
 
+.. note:: Two Wi-Fi USB modules have been tested with KA Lite on the Raspberry Pi
 
-Developer setup
-_______________
+    * Raspberry Pi WiPi adaptor
+    * Edimax EW-7811Un
 
-Developers should consider installing in "editable" mode. That means, create a
-git clone and from the git cloned directory, run::
+In our tests, we found that the WiPi adaptor supported a higher number tablet connections.
 
-    $> git clone git@github.com:learningequality/ka-lite.git
-    $> cd ka-lite
-    $> # You may wish to create and activate a virtual env here
-    $> pip install -e .
 
+.. note:: The Raspberry Pi may crash if the USB adaptor is inserted or removed while the computer is switched on.
 
-Testing installers
-__________________
+    * Make sure to shutdown and remove the power from the Raspberry Pi.
+    * Afterwards, insert the wireless USB adaptor.
+    * Lastly, switch the Raspberry Pi on.
 
-Here's an overview of the various ways of installing KA Lite as a reference
-to testers and package maintainers:
+#. Install the .deb package: ``dpkg -i /path/to/ka-lite-raspberry-pi.deb``

```

Comment:
Fixed, good catch! replaced with cross-ref for installation section where that was actually missing!!